image_verification/check_attestations.yaml

```diff
@@ -42,10 +43,7 @@ spec:
               - key: "{{ scanner }}"
                 operator: Equals
                 value: trivy
-              - key: "{{ time_since('', timestamp, '') }}"
-                operator: LessThan
-                value: 24h
-              - key: "{{ Results[].Vulnerabilities[?CVSS.redhat.V3Score > `8.0`][] | length(@) }}"
+              - key: "{{ Results[].Vulnerabilities[?CVSS.redhat.V3Score >= `10.0`][] | length(@) }}"
```

Comment:
For the demo, its nice to typically show that a resource with an older scan is blocked and allow a resource with an up to date scan to be allowed.

Same for the CVSS score.

Should we have two policies, or maybe have a way to patch the existing policy via kubectl?

Reply:
Added a test with 2 diff. images.